converter/output/rita/freqconn/counter.go

```diff
@@ -0,0 +1,82 @@
+package freqconn
```

Comment:
ConnCounter holds the count map for each connection pair. It calls out to a ConnCountNotifier when a connection count meets or exceeds a given threshold. This separation of concerns made it much easier to test the code. The StrobesNotifier implements ConnCountNotifier and implements the MongoDB logic to handle strobes.